spec/features/volunteer_schedule_absence_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'Scheduling an Absence' do
+
+  let(:volunteer_with_assignment) { create :volunteer_with_assignment }
+  let(:volunteer_without_assignment) { create :volunteer_with_region }
+  let(:admin) { create :volunteer_with_assignment, admin: true }
+
+    context 'non-admin user' do
+      context 'WITHOUT assignment' do
+        it 'cannot schedule' do
+          login volunteer_without_assignment
+
+           within '.navbar' do
+             click_on 'Schedule An Absence'
+           end
+           
+           expect(current_path).to eq(new_absence_path)
+
+           click_on 'Save changes'
+
+           expect(current_path).to eq(absences_path)
+           within('.alert') do
+            expect(page).to have_content('No shifts of yours was found in that timeframe')
+           end
+        end
+      end
+    end
+end
```

Comment:
The indentation is a little funky in this file. It looks like lines 9-28 are indented by two extra spaces and lines 14-23 and line 25 are indented by an extra space.